sourcegraph/sourcegraph
# admin-analytics: change "Unique users / Active users" to use registered users count instead of anonymous count

Closes https://github.com/sourcegraph/sourcegraph/issues/41245.

## Test plan
- check the diff
- `sg start`
- Open http://localhost:3080/site-admin and the rest of the analytics feature pages
- Check that "total/unique" charts load without any errors



## App preview:

- [Web](https://sg-web-erzhtor-fix-admin-analytics-to.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-vqzugdxwys.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
